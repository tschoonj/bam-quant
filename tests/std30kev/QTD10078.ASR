File: C:\xrfdaten\2014KW30\chris\std30kev\std10078.spe
Start: 22.07.2014 10:26:18
Stop: 22.07.2014 10:27:58
BESSY: 298.9616035 297.9503171
IONI: 1.949303E-11 1.942197E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97417
Run: 100000
Dead: 3
ICR: 2480
OCR: 0
Accumulated BESSY: 59690.8442704
Accumulated Ioni: 3.89038102E-09
N Accumulation: 200
Normfactor:  5.2771946e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pt
exp_x 92.7203979492188 92.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  78   2      9.434     18234.       135.     126.50
