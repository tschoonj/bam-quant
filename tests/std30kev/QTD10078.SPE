$DATA:
0 2000
23052
14530
6179
1854
333
42
8
0
1
0
0
0
0
0
0
0
12
62
68
77
78
69
71
64
63
48
47
61
57
48
54
49
52
46
39
44
37
39
51
41
36
40
37
44
36
47
36
31
27
27
40
35
35
27
34
25
38
31
27
38
33
20
36
27
31
28
26
15
28
30
28
30
38
41
36
23
32
21
27
18
17
19
24
28
21
19
22
25
27
20
29
26
19
26
32
31
25
17
18
23
26
32
26
25
14
39
24
14
26
25
25
20
15
28
21
25
20
13
18
18
32
19
17
20
24
28
12
21
24
12
19
21
17
17
18
23
17
11
17
12
17
25
18
14
18
22
19
14
14
22
12
18
13
19
18
16
19
25
18
18
18
13
15
17
23
25
28
19
22
17
20
17
20
18
17
21
11
20
12
17
18
24
11
16
22
20
17
14
12
24
16
21
13
13
24
18
19
16
13
15
18
16
22
8
13
18
19
14
12
20
16
13
11
16
12
21
17
19
20
13
21
15
14
20
11
14
15
14
13
16
12
9
16
9
19
18
13
15
16
14
12
13
12
12
20
11
16
17
17
13
14
14
13
14
12
23
15
16
11
10
16
9
23
8
20
16
16
11
16
12
15
13
16
14
17
10
19
11
11
12
8
9
13
10
12
12
10
13
15
10
10
9
14
10
19
15
16
13
8
6
12
4
14
15
7
11
13
10
10
9
13
7
14
13
14
22
28
27
20
11
18
9
10
12
11
12
6
12
9
13
14
13
7
7
7
9
14
14
16
16
13
24
13
11
9
14
13
12
9
16
17
16
10
18
13
15
7
14
15
18
12
18
14
9
10
22
14
19
22
12
14
13
11
12
11
21
15
12
14
20
17
14
19
22
20
14
16
17
18
9
14
23
6
12
14
7
16
13
15
25
23
21
17
16
24
22
24
27
22
23
32
31
24
33
21
18
23
22
20
18
17
13
21
15
14
24
23
19
29
14
22
16
20
14
21
10
20
24
13
20
21
26
21
19
21
22
24
22
24
31
24
23
40
44
55
50
77
80
153
244
359
563
843
1126
1548
1815
2075
2163
2146
2035
1805
1469
1095
700
476
298
172
115
86
60
55
41
32
33
37
40
42
59
42
59
71
92
103
124
119
122
108
103
81
72
68
49
46
38
48
41
37
35
33
47
37
42
47
59
56
54
40
64
59
62
57
68
63
72
78
111
98
128
188
222
303
445
642
811
977
1205
1416
1523
1734
1856
2015
2430
3276
4151
5518
6775
8060
9026
9369
9617
9131
8553
7582
7087
6640
6219
6102
5758
5350
4729
3952
3226
2454
1818
1289
893
694
508
419
349
326
315
304
299
284
282
240
194
188
161
123
133
135
105
78
75
57
57
41
35
26
28
28
24
28
19
32
28
19
17
31
22
30
19
29
21
29
26
24
32
34
28
40
39
48
60
79
98
80
108
130
117
116
120
103
91
92
91
107
120
168
189
301
447
690
1002
1323
1710
1994
2317
2503
2477
2458
2162
1962
1529
1214
889
696
542
502
496
544
636
790
935
1016
1125
1189
1153
1147
1073
975
765
646
517
403
347
230
148
113
106
78
80
70
67
65
75
95
122
156
170
201
189
214
205
195
184
175
115
87
70
51
37
28
20
15
18
16
7
10
8
8
8
11
12
10
13
13
18
11
7
10
9
8
11
12
11
6
7
14
12
13
6
10
13
6
10
12
7
9
6
10
11
10
8
10
13
8
17
15
9
15
14
11
10
9
9
10
8
11
17
5
13
15
7
15
5
11
13
13
10
8
11
11
8
11
6
15
16
16
29
32
34
34
49
68
64
74
77
80
69
86
64
59
49
37
42
29
19
11
16
8
19
6
16
7
7
13
13
15
9
14
5
11
7
13
14
7
13
12
10
8
6
8
4
5
12
8
15
6
12
12
9
8
10
16
6
16
15
8
6
10
9
14
9
10
9
18
13
8
11
4
10
9
14
11
10
17
12
12
9
10
9
11
6
8
11
11
12
8
12
12
10
18
9
11
14
12
17
20
12
21
19
29
21
24
19
28
22
11
13
16
16
14
5
17
20
8
10
8
12
13
14
14
16
13
8
18
7
7
7
5
12
11
6
6
20
10
8
18
13
13
7
11
16
9
13
8
19
12
16
11
9
13
9
10
9
10
6
7
13
7
9
13
8
6
8
14
5
12
8
9
11
10
8
8
12
7
5
13
11
8
4
10
9
12
10
8
3
8
7
6
8
9
8
6
11
9
9
15
8
9
14
6
11
11
5
13
14
11
10
10
7
16
11
11
11
13
6
9
8
11
5
10
14
7
14
7
5
13
10
8
16
6
10
3
11
5
13
5
7
5
7
8
8
11
5
4
9
7
8
9
7
9
6
4
10
13
7
12
12
3
9
7
6
2
4
5
5
3
9
7
10
8
4
9
6
7
11
8
6
9
7
7
10
5
8
9
7
4
7
4
9
7
5
14
8
6
7
6
6
5
3
7
4
5
0
6
6
6
5
8
6
8
6
2
9
4
3
6
8
8
11
8
6
9
0
2
3
6
8
9
4
3
7
3
6
4
9
7
10
5
9
10
5
7
5
5
4
6
2
3
7
7
4
6
12
3
8
4
6
2
7
4
4
2
3
4
5
3
5
6
5
4
3
9
9
2
5
3
3
3
4
4
2
2
3
3
3
3
1
3
2
2
1
3
2
4
4
4
4
3
1
2
1
6
4
4
5
6
5
5
5
5
8
3
3
4
4
2
4
1
0
3
3
9
5
4
2
6
6
4
5
2
1
4
6
6
2
2
2
2
1
2
1
1
5
3
4
2
6
1
2
7
3
2
1
6
2
5
3
4
5
2
5
8
10
7
4
7
4
2
7
14
9
10
12
8
5
7
8
7
10
5
8
6
7
7
7
7
4
3
4
1
1
5
2
1
3
4
4
4
3
5
1
1
1
5
2
2
0
2
6
4
5
5
4
2
2
3
3
2
2
3
5
2
5
7
5
3
1
4
5
5
6
2
3
4
2
7
3
6
5
3
1
2
1
2
0
3
3
1
3
2
2
2
4
5
4
1
1
1
0
3
5
1
3
4
1
2
7
0
2
1
4
5
5
5
3
5
4
4
7
2
1
5
6
3
7
4
4
6
4
2
5
4
1
2
7
5
2
2
2
5
2
1
3
1
0
0
2
2
3
2
2
2
1
2
1
3
1
3
1
4
5
2
4
1
1
1
2
2
4
4
1
4
4
3
7
5
3
3
5
5
4
6
1
4
7
0
0
7
3
1
3
2
6
2
4
5
1
3
1
3
3
4
2
3
3
3
2
2
2
7
3
3
5
2
3
6
4
5
1
3
5
2
3
2
5
3
2
2
2
0
3
3
1
6
0
1
3
2
4
3
5
5
1
0
3
1
3
2
4
5
5
1
3
5
2
2
4
3
4
2
4
4
4
3
2
5
3
0
5
3
6
3
5
3
3
5
3
3
6
7
5
2
5
4
8
4
6
5
4
2
6
6
4
2
3
4
4
8
2
2
9
6
2
7
4
8
3
7
5
5
4
5
4
5
5
5
7
5
5
4
3
2
7
9
8
5
6
4
7
9
10
7
7
8
4
5
8
2
8
8
7
5
7
8
7
5
5
6
6
10
7
12
8
5
6
11
9
6
9
11
8
9
8
4
10
4
8
7
10
13
9
12
13
5
6
7
8
13
7
10
9
8
9
9
12
13
13
5
3
12
11
13
9
20
10
9
5
9
10
8
10
8
10
7
7
9
9
9
2
8
15
12
6
9
12
16
7
8
9
12
15
10
15
9
7
14
8
12
11
15
9
11
14
17
12
10
9
12
10
11
12
8
12
19
12
16
7
12
17
20
8
18
12
14
20
23
15
15
11
14
16
15
9
14
16
16
19
8
16
20
7
15
18
15
18
14
17
16
18
21
23
23
30
43
41
41
53
52
54
64
58
81
85
89
88
104
109
77
116
94
97
93
99
88
104
98
80
100
96
72
100
98
87
86
91
67
74
65
53
62
56
45
47
31
28
21
21
20
26
21
15
9
9
8
6
6
6
0
2
1
2
4
1
0
0
0
1
2
0
0
1
1
0
0
2
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10078.spe
Start: 22.07.2014 10:26:18
Stop: 22.07.2014 10:27:58
BESSY: 298.9616035 297.9503171
IONI: 1.949303E-11 1.942197E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97417
Run: 100000
Dead: 3
ICR: 2480
OCR: 0
Accumulated BESSY: 59690.8442704
Accumulated Ioni: 3.89038102E-09
N Accumulation: 200
Normfactor:  5.2771946e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pt
 
exp_x 92.7203979492188 92.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  78   2      9.434     18234.       135.     126.50
