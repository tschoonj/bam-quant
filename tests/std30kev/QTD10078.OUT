     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:59  บ   
     บ  Spectrum: QTD10078.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        3 iterations done    บ   
     บ  ChiSquare =    14.1   last change =   .04%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -68.7 ๑   100.0   บ     -68.7 ๑     2.7   บ   
     บ  GAIN (eV/ch)  บ     20.248 ๑   2.000   บ    20.248 ๑    .005   บ   
     บ det NOISE (eV) บ      182.1 ๑    40.0   บ     182.1 ๑     7.7   บ   
     บ  FANO factor   บ       .055 ๑    .050   บ      .055 ๑    .008   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Pt-L1                             59761. ๑    351.           บ   
     บ    L1M3       11.231    .39992       23900. ๑    141.    223.9  บ   
     บ              558.056    215.22         229.           9.83E-01  บ   
     บ    L1M2       10.854    .33846       20227. ๑    120.     96.3  บ   
     บ              539.438    214.19         210.           9.82E-01  บ   
     บ    L1N3       13.361    .11195        6690. ๑     42.     28.6  บ   
     บ              663.250    220.93         207.           9.87E-01  บ   
     บ    L1N2       13.270    .08745        5226. ๑     34.     21.6  บ   
     บ              658.756    220.69         208.           9.86E-01  บ   
     บ    L1O3       13.822    .03448        2060. ๑     19.      6.2  บ   
     บ              686.017    222.15         201.           9.86E-01  บ   
     บ    L1M5       11.758    .01663         994. ๑     16.     39.0  บ   
     บ              584.083    216.64         224.           9.85E-01  บ   
     บ    L1M4       11.676    .01112         664. ๑     15.     75.5  บ   
     บ              580.034    216.42         225.           9.85E-01  บ   
     บ                                                                 บ   
     บ  2 Pt-L2                            127592. ๑    437.           บ   
     บ    L2M4       11.071    .79220      101079. ๑    347.    136.6  บ   
     บ              550.155    214.78         231.           9.83E-01  บ   
     บ    L2N4       12.943    .16358       20872. ๑     73.    104.0  บ   
     บ              642.607    219.82         211.           9.86E-01  บ   
     บ    L2M1        9.975    .02146        2738. ๑     17.    103.5  บ   
     บ              496.027    211.77         218.           9.77E-01  บ   
     บ    L2N1       12.552    .00561         715. ๑     15.     18.7  บ   
     บ              623.296    218.78         215.           9.84E-01  บ   
     บ    L2O4       13.271    .01525        1946. ๑     16.     21.6  บ   
     บ              658.805    220.70         208.           9.86E-01  บ   
     บ    L2O1       13.173    .00110         141. ๑     14.     30.6  บ   
     บ              653.966    220.44         209.           9.86E-01  บ   
     บ    L2M3       10.627    .00080         102. ๑     15.     26.1  บ   
     บ              528.227    213.57         212.           9.81E-01  บ   
     บ                                                                 บ   
     บ  3 Pt-L3                             22946. ๑    170.           บ   
     บ    L3M5        9.442    .71362       16375. ๑    122.    126.5  บ   
     บ              469.704    210.29         223.           9.73E-01  บ   
     บ    L3M4        9.362    .08105        1860. ๑     20.     78.6  บ   
     บ              465.753    210.07         223.           9.72E-01  บ   
     บ    L3N5       11.251    .14054        3225. ๑     28.    228.9  บ   
     บ              559.044    215.27         229.           9.84E-01  บ   
     บ    L3M1        8.268    .03904         896. ๑     17.     84.8  บ   
     บ              411.723    207.00         233.           9.60E-01  บ   
     บ    L3N1       10.842    .00978         224. ๑     15.     95.1  บ   
     บ              538.845    214.15         210.           9.82E-01  บ   
     บ    L3O1       11.462    .00192          44. ๑     15.    259.5  บ   
     บ              569.465    215.84         227.           9.84E-01  บ   
     บ    L3O5       11.561    .01405         322. ๑     15.    120.2  บ   
     บ              574.354    216.11         226.           9.85E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                            -108. ๑     17.           บ   
     บ    L3O5       22.142    .25568         -28. ๑     12.      2.4  บ   
     บ             1096.915    243.09         118.           8.99E-01  บ   
     บ    L3O1       22.304    .13809         -15. ๑     11.      2.3  บ   
     บ             1104.932    243.48         116.           8.94E-01  บ   
     บ    L3O5       24.014    .10559         -11. ๑     10.      1.3  บ   
     บ             1189.367    247.56          96.           8.39E-01  บ   
     บ    L2N4       21.925    .10437         -11. ๑     11.      2.8  บ   
     บ             1086.192    242.57         120.           9.05E-01  บ   
     บ    L2M1       20.505    .09095         -10. ๑     12.      1.1  บ   
     บ             1016.061    239.12         138.           9.41E-01  บ   
     บ    L2N1       24.384    .07139          -8. ๑     10.      2.3  บ   
     บ             1207.626    248.43          92.           8.26E-01  บ   
     บ    L3M5       24.176    .02851          -3. ๑     10.      1.8  บ   
     บ             1197.384    247.94          94.           8.33E-01  บ   
     บ    L3N5       22.087    .02819          -3. ๑     11.      2.4  บ   
     บ             1094.209    242.96         118.           9.00E-01  บ   
     บ    L3M1       20.667    .02456          -3. ๑     12.      1.4  บ   
     บ             1024.078    239.52         136.           9.37E-01  บ   
     บ    L3N1       23.797    .02155          -2. ๑     10.      1.8  บ   
     บ             1178.644    247.05          99.           8.46E-01  บ   
     บ               24.546    .01928          -2. ๑      9.      2.7  บ   
     บ             1215.643    248.82          90.           8.21E-01  บ   
     บ               22.377    .01878          -2. ๑     11.      1.8  บ   
     บ             1108.513    243.66         115.           8.92E-01  บ   
     บ               22.467    .01865          -2. ๑     11.      1.5  บ   
     บ             1112.949    243.87         114.           8.89E-01  บ   
     บ               20.288    .01856          -2. ๑     12.      1.7  บ   
     บ             1005.338    238.59         140.           9.45E-01  บ   
     บ               26.256    .01474          -2. ๑      8.       .9  บ   
     บ             1300.078    252.81          72.           7.59E-01  บ   
     บ               24.167    .01457          -2. ๑     10.      1.8  บ   
     บ             1196.902    247.92          94.           8.34E-01  บ   
     บ               21.046    .01385          -1. ๑     11.      1.5  บ   
     บ             1042.787    240.44         131.           9.28E-01  บ   
     บ               22.747    .01270          -1. ๑     11.      1.2  บ   
     บ             1126.772    244.55         111.           8.81E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
